CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.4.2 (October 17, 2020)
+
+- Warn if a config is undefined
```

Comment:
I'd add that you're also warning on empty string and update the README to mention this as well.